### DISCUSSION TITLE
aria的文件预分配改为NONE

### DISCUSSION THREAD

**Discussion by ahlarry** [leiurayer/downkyi#295] [General]
这项更新有什么好处，还是单纯的只是缩短预分配时间？

**Comment by leiurayer** [comment]
设置文件预分配，可以降低磁盘碎片，
预分配所需时间：none < falloc ? trunc < prealloc

> Specify file allocation method. none doesn't pre-allocate file space. prealloc pre-allocates file space before download begins. This may take some time depending on the size of the file. If you are using newer file systems such as ext4 (with extents support), btrfs, xfs or NTFS(MinGW build only), falloc is your best choice. It allocates large(few GiB) files almost instantly. Don't use falloc with legacy file systems such as ext3 and FAT32 because it takes almost same time as prealloc and it blocks aria2 entirely until allocation finishes. falloc may not be available if your system doesn't have posix_fallocate(3) function. trunc uses ftruncate(2) system call or platform-specific counterpart to truncate a file to a specified length.
